### ISSUE TITLE
Add clearer indication to the user in the hotkey settings window when hotkeys are disabled

### ISSUE THREAD

**Issue by cesarmalari** [ShareX/ShareX#2122]
I filed #2115 yesterday because I managed to disable hotkeys by accident and didn't realize it.  The only clue I had was the color of the boxes on the right side of the screen, which I thought to click on/mouse over, but they don't have any tooltips on them either.

I'd recommend adding tooltips to those boxes, plus some kind of notification bar at the top or bottom of the screen to let the user know that nothing they do in that screen means anything if they don't enable the hotkeys.

**Comment by bartinfinity** [comment]
I had this issue since 26th February this year (at least it is last screenshot I made). Later on somehow ShareX stopped taking screenshots from shortcuts and I had page with shortcuts with yellow light that I didn't know what it means. I thought it is some kind of issue with permissions or registering keys. I just today so 30th April discovered your post and that helped me a lot.  I don't know when or how I disabled hotkeys, I didn't saw any hotkey to disable hotkeys so it had to be my mistake when I opened ShareX menu and I didn't realized. More over you reported this in 2016 and now is 2021 and they only added something like that? Some small message that disappear after few seconds that is actually not untestable enough?
![chrome_2021-04-30_18-24-11](https://user-images.githubusercontent.com/2501014/116725061-bb4c2f00-a9e1-11eb-812f-e71f6e381f26.png)
Because whole window with hotkeys didn't gave my any clue that hotkeys are disabled at all so I had window with list of hotkeys and yellow light like this:
![2021-04-30_18-28-49](https://user-images.githubusercontent.com/2501014/116725197-eb93cd80-a9e1-11eb-836c-8f25a47ebbbd.png)
I think this still needs "enhancement" please put it again in enhancement list and truly leave a note that hotkeys are disabled, it can be in hotkey window somewhere below, it should be simple if logic that would return some simple text that anyone would understand why their hotkeys are not working no matter what they do.

**Comment by bartinfinity** [comment]
@Jaex yes, how to get it? Is it in some beta version? Also I wonder why you have all hotkeys with green light, I thought when they are disabled they get yellow light.

**Comment by bartinfinity** [comment]
> @bartinfinity like this?
> 
> ![](https://camo.githubusercontent.com/bb76911832fd2a014e6cca83ab2088a5c8c0f2877ba2920d785287cfc29e67f3/68747470733a2f2f6a6165782e6765747368617265782e636f6d2f323032312f30342f5368617265585f6943707961586e576e742e706e67)

Cool cool, works very good! I hope that will helpy someone who by accidentally disable hotkeys like me :)

![2021-04-30_20-30-54 (2)](https://user-images.githubusercontent.com/2501014/116738778-2eaa6c80-a9f3-11eb-9aa0-ad1d8fafde91.png)
